test/test_helper.rb

```diff
@@ -5,12 +5,15 @@
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'ttnt'
+require 'helpers/git_helper'
+require 'helpers/rake_helper'
 
+require 'rugged'
 require 'minitest/autorun'
 
 module TTNT
   class TestCase < Minitest::Test
-    FIZZBUZZ_FIXTURE_DIR = File.expand_path('../fixtures/repositories/fizzbuzz', __FILE__).freeze
+    FIXTURE_DIR = File.expand_path('../fixtures', __FILE__).freeze
```

Comment:
Actually the `freeze` call isn't necessary here I guess ; you are already storing the value in a constant. Also, you can simply set it to `File.join(__dir__, 'fixtures')`.